pandas/core/arrays/categorical.py

```diff
@@ -447,7 +447,12 @@ def __init__(
             if isinstance(values.dtype, ArrowDtype) and issubclass(
                 values.dtype.type, CategoricalDtypeType
             ):
-                arr = values._pa_array.combine_chunks()
+                from pandas import Index
+
+                if isinstance(values, Index):
+                    arr = values._data._pa_array.combine_chunks()
+                else:
+                    arr = values._pa_array.combine_chunks()
```

Comment:
```suggestion
                arr = values.array._pa_array.combine_chunks()
```

Reply:
Ah ok. It appears something is passing in an `ArrowExtensionArray` here already. OK what you have here is fine then